### PR TITLE
Remove Stories and Kanvas dependency

### PR DESCRIPTION
**Pull request by kean**
The "Stories" feature was [decommissioned](https://github.com/wordpress-mobile/WordPress-iOS/pull/22449) five months ago: you can no longer created, view, or edit stories natively in the app. This PR removes the remaining code.

- Reduces the universal .ipa size by another **2.7 MB**
- Removes the `Kanvas` dependency (25K of Swift) – one less pod to deal with

## To test:

- Open the app and verify that it still runs
- Open "Site Media" and verify that the cells are displayed

## Regression Notes
1. Potential unintended areas of impact: n/a
2. What I did to test those areas of impact (or what existing automated tests I relied on): n/a

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/Classes/ViewRelated/Media/External/ExternalMediaPickerViewController.swift [hunk 1]
```diff
@@ -86,7 +88,7 @@ final class ExternalMediaPickerViewController: UIViewController, UICollectionVie
     }
 
     private func configureCollectionView() {
-        collectionView.register(cell: ExternalMediaPickerCollectionCell.self)
+        collectionView.register(ExternalMediaPickerCollectionCell.self, forCellWithReuseIdentifier: Self.cellReuseID)
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Media/External/ExternalMediaPickerViewController.swift:91]: This method was coming from `Kanvas`. I'm not sure why it was working without the `import` – pch, ObjC briding header?

File: WordPress/Classes/ViewRelated/Reader/Theme/ReaderDisplaySetting.swift [hunk 2]
```diff
@@ -434,3 +434,49 @@ class ReaderDisplaySettingStore: NSObject {
...
 }
+
+private extension UIColor {
+    /**
+    Whether or not the color brightness is higher than a provided brightness value.
+
+    - parameter brightnessValue: A number that represents the brightness of a color. It ranges from 0.0 (black) to 1.0 (white).
+    - return: YES if brightness is higher than the brightness value provided.
+    */
+    public func brighterThan(_ brightnessValue: Double) -> Bool {
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Reader/Theme/ReaderDisplaySetting.swift:445]: Copied form Kanvas.

File: WordPress/Classes/ViewRelated/Blog/QuickStartChecklistManager.swift [hunk 3]
```diff
@@ -63,7 +65,7 @@ extension QuickStartChecklistManager: UITableViewDelegate {
     }
 
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: QuickStartChecklistHeader.identifier) as? QuickStartChecklistHeader
+        let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: QuickStartChecklistHeader.defaultReuseID) as? QuickStartChecklistHeader
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Blog/QuickStartChecklistManager.swift:68]: The registration code looks like this:

```
tableView.register(QuickStartChecklistHeader.defaultNib, forHeaderFooterViewReuseIdentifier: QuickStartChecklistHeader.defaultReuseID)
```

The `identifier` was coming from `Kanvas` and its `Identifiable` protocol.

**Review by jkmassel — APPROVED** [review]

**Pull request merged**